src/components/Map.vue

```diff
@@ -171,6 +190,37 @@ function initiateMap(source: VectorSource<Geometry>) {
       maxZoom: 19,
       showFullExtent: true,
     }),
+    interactions: defaultInteractions({
+      dragPan: false,
+      mouseWheelZoom: false,
+      altShiftDragRotate: false,
+      pinchRotate: false,
+    }).extend([
+      new DragPan({
+        condition: function (event) {
```

Comment:
This is a nitpick. I know a lot of people in this oganization prefer to use arrow functions for anonymous functions. i.e. 
```suggestion
        condition: (event) => {
```
Personally I don't care, but for the sake of consistency it might make sense to change.

Reply:
For some, unknown, reason it just straight up does not work if using arrow functions instead